packages/runtime/src/runtime/app/botpress.ts

```diff
@@ -197,6 +197,7 @@ export class Botpress {
 
     await this.cmsService.initialize()
     await this.eventCollector.initialize(this.database)
+    await this.messagingService.initialize()
```

Comment:
Actualy this should be added elsewhere. There are two different "setup" for the runtime: either embedded (when executed by botpress server) or standalone (when executed in the cloud).

On line 103, messaging is initialized by botpress server (since we might get the messaging port later during initialization).

I suggest you move this line in "initStandalone" after the service initialization.

Most of my tests were done using the embedded version since it was easier to test it